src/Type/BasePickerType.php

```diff
@@ -60,39 +63,24 @@ public function __construct(MomentFormatConverter $formatConverter, $translator
             ));
         }
 
-        if (null === $requestStack) {
-            if ($translator instanceof TranslatorInterface) {
-                throw new \InvalidArgumentException(sprintf(
-                    'Argument 3 should be an instance of %s',
-                    RequestStack::class
-                ));
-            }
-
-            @trigger_error(sprintf(
```

Comment:
This will trigger deprecation if you remove the request stack from the services file?

Reply:
RequestStack is in 1.x. We should make 0.x(backport) as much as possible to look like CoreBundle part(form directory). You should check diff with CoreBundle, not form-extensions 1.x. Diff with 1.x we should add to Upgrade-1.0 -1.x.

Make CoreBundle optional process:
- Upgrade CoreBundle to 3.20
- add -extensions 0.x to composer.json and register bridges
- drop CoreBundle
- upgrade extensions to 1.x